helm_deploy/hmpps-prisoner-to-nomis-update/templates/report-adjudications-cronjob.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-report-sentencing
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "30 5 * * 2"
+  suspend: true
```

Comment:
disabled for now

Reply:
Maybe make it a wednesday as the NA recon runs on Tues?

Reply:
yes, will do - was a bit random until I really schedule it in